src/test/java/com/mabl/integration/jenkins/MablRestApiClientTest.java

```diff
@@ -45,12 +43,12 @@
     private static final String fakeProperties = "{\"deployment_origin\":\""+MablStepConstants.PLUGIN_USER_AGENT+"\"}";
 
     @Test
-    public void createDeploymentAllParametersHappyPathTest() throws IOException, MablSystemError {
+    public void createDeploymentAllParametersHappyPathTest() throws IOException, MablSystemException {
```

Comment:
This is what I was referring to with checked vs unchecked exception. We are having to declare them but it does not really add much to code clarity or API usability.